src/content/docs/en/recipes/bun.mdx

```diff
@@ -2,19 +2,100 @@
+:::caution
+Bun recently made their first stable release. However, using Bun with Astro may reveal rough edges. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details.
 
 If you have any problems using Bun, please [open an Issue on GitHub with Bun directly](https://github.com/oven-sh/bun/issues/new/choose).
+:::
 
-## Community Resources
+## Prerequisites
 
-Using Bun with Astro? Add your blog post or video to this page!
+- Bun installed locally on your machine. See the [installation instructions](https://bun.sh/docs/installation) in Bun's official documentation.
+
+## Create a new Astro project with Bun
+
+Create a new Astro project with Bun using the following `create-astro` command:
+
+```bash
+bunx create-astro@latest my-astro-project-using-bun
+```
+
+:::tip
+You may also [create a new Astro project from any existing Astro GitHub repository](/en/install/auto/#starter-templates) using the `--template` flag:
+```bash
+bunx create-astro@latest my-astro-project-using-bun --template eliancodes/brutal
+```
+:::
+
+## Install dependencies
+
+If you're starting a new project using `bunx create-astro`, the CLI will automatically use Bun to install dependencies and you can skip this step.
+
+Otherwise, you'll need to install your dependencies with Bun:
+
+```bash
+bun install
+```
+
+## Using Astro integrations
+
+You can also use any of the official Astro integrations with the `astro add` command:
+
+```bash
+bunx astro add react
+```
+
+## Run Astro with Bun
+
+Use the [`--bun` CLI flag](https://bun.sh/docs/cli/bunx#shebangs) to tell Bun to use it's runtime in place of Node for the `astro` command.
+
+### Run the development server
 
-[Building a Cloudflare Pages site with Bun](https://blog.otterlord.dev/posts/hello-from-bun/) - blog post
+To run the development server, using Bun as the runtime, use the following command:
 
+```bash
+bun --bun astro dev
+```
+
+### Building your site with Bun
+
+To build your site, using Bun as the runtime, use the following command:
+
+```bash
+bun --bun astro build
+```
+
+Astro will output your site to the `dist/` directory. Then, you can serve your site using the `preview` command:
+
+```bash
+bun --bun astro preview
+```
+
+## Add SSR with Bun using `@astrojs/node`
+
+Bun features [Node.js API compatibility](https://bun.sh/docs/runtime/nodejs-apis). Using the [`@astrojs/node`](/en/guides/integrations-guide/node/) adapter you can add [server-side rendering](/en/guides/server-side-rendering/) to your Astro project, using Bun's runtime as a replacement for Node.
+
+```bash
+bunx astro add node
+```
+
+Once you build a project with `bun --bun astro build`, use the following to run your built site.
+
```

Comment:
``````suggestion
Run the following command to add the Node.js adapter to your Astro project:

```bash
bunx astro add node
```

Build your site again, using the same build command as above:

```bash
bun --bun astro build
```
Finally, use the following command to run your built site.

``````
Does this work?